packages/ui/src/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,51 @@
+import type { ReactNode } from "react";
+import * as DropdownMenu from "@radix-ui/react-dropdown-menu";
+import type { DropdownMenuContentProps } from "@radix-ui/react-dropdown-menu";
+import clsx from "clsx";
+
+// ==============================
+// Radix Components
+// https://github.com/radix-ui/primitives/blob/main/packages/react/dropdown-menu/src/index.ts
+// ==============================
+export const Root = DropdownMenu.Root;
+export const Trigger = DropdownMenu.Trigger;
+export const Portal = DropdownMenu.Portal;
+export const Group = DropdownMenu.Group;
+export const Label = DropdownMenu.Label;
+export const Item = DropdownMenu.Item;
+export const CheckboxItem = DropdownMenu.CheckboxItem;
+export const RadioGroup = DropdownMenu.RadioGroup;
+export const RadioItem = DropdownMenu.RadioItem;
+export const ItemIndicator = DropdownMenu.ItemIndicator;
+export const Separator = DropdownMenu.Separator;
+export const Arrow = DropdownMenu.Arrow;
+export const Sub = DropdownMenu.Sub;
+export const SubTrigger = DropdownMenu.SubTrigger;
+export const SubContent = DropdownMenu.SubContent;
```

Comment:
You have a point there. Maybe they shouldn't.

If you need some new version of the DropdownMenu in apps/extension -- how would you create it? Would you:
- import the components from radix in apps/extension to stitch it together,
- create a new variant in packages/ui to import it into apps/extension when ready
?

I'd probably do the first one just because this new design needs to fit my use case (apps/extension) and it's sometimes a bit hard for me to understand how I should make my pre-defined `Dialog` from ui fit for this new use case without breaking the old ones.

That may be because I don't have a lot of experience maintaining design systems, so happy to adopt a best practice if there is one.